app/src/components/widgets/charts/overallStatisticsChart/overallStatisticsPanel.jsx

```diff
@@ -26,17 +26,33 @@ export class OverallStatisticsPanel extends React.PureComponent {
     const { widget, orderedContentFields } = this.props;
     const values = widget.content.result[0].values;
 
-    return orderedContentFields.map((key) => ({ key, value: values[key] || 0 }));
+    return orderedContentFields
+      .filter((key) => widget.contentParameters.contentFields.indexOf(key) !== -1)
+      .map((key) => ({ key, value: values[key] || 0 }));
   };
 
-  render() {
-    const { widget, isPreview } = this.props;
+  getTotals = () => {
+    const { widget } = this.props;
     const values = widget.content.result[0].values;
+    const fields = widget.contentParameters.contentFields.filter(
+      (field) => field.indexOf('executions') !== -1,
```

Comment:
I think you should also filter values by their selected/unselected status to support the following case from EPMRPP-38910 (fields should be hidden ONLY in case when user deselects them in the widget settings). This case now is broken by changes in this PR.

Reply:
When user unselects some values, those values will automatically disappear from `widget.contentParameters.contentFields` array and don't be displayed on the widget, so I think it's redundant to additionally filter them. If server doesn't return some values, but user selects them to display, widget shows zeroes (line 43).

Reply:
In case when field exists in `contentFields`, but doesn't exists in `values` it shows incorrectly:
![image](https://user-images.githubusercontent.com/2759038/59771362-37d43980-92b2-11e9-9049-374e3b6dff02.png)

JSX conditions like `{passed && ...` in case `passed = 0` render as `0`.